pre-registration/pre-registration-core/src/main/java/io/mosip/analytics/event/anonymous/service/AnonymousProfileService.java

```diff
@@ -67,9 +70,9 @@ public AnonymousProfileResponseDTO saveAnonymousProfile(AnonymousProfileRequestD
 			AnonymousProfileEntity requestEntity = new AnonymousProfileEntity();
 			requestEntity.setId(UUIDGeneratorUtil.generateId());
 			requestEntity.setProfile(requestDto.getProfileDetails());
-			requestEntity.setCreatedBy(authUserDetails().getUserId());
+			requestEntity.setCreatedBy(userName);
```

Comment:
we are hardcoding to admin name here, instead should we not have some salted hash of the username ? Not sure if this can be helpful for some statistics?

Reply:
We had discussion with BA and came to conclusion to have a default name (admin) as to make it anonymous